bot/bot.py

```diff
@@ -15,9 +15,9 @@ def load_extensions(self) -> None:
         """Load all the extensions in the exts/ folder."""
         logger.info('Start loading extensions from ./exts/')
         for extension in constants.EXTENSIONS.glob("*/*.py"):
+            if extension.name[0] == "_":
+                continue  # ignore files starting with _
```

Comment:
Would it be more pythonic to do this?
```suggestion
            if extension.name.startswith("_"):
                continue  # ignore files starting with _
```

Reply:
right of course,